main_test.go

```diff
@@ -72,3 +75,64 @@ func (t *MainTest) TestGetUserAgentWhenMetadataImageTypeEnvVarSetAndAppNameNotSe
 
 	ExpectEq(expectedUserAgent, userAgent)
 }
+
+func (t *MainTest) TestStringifyShouldReturnAllFlagsPassedInMountConfigAsMarshalledString() {
+	mountConfig := &config.MountConfig{
+		WriteConfig: config.WriteConfig{
+			CreateEmptyFile: false,
+		},
+		LogConfig: config.LogConfig{
+			Severity: config.TRACE,
+			FilePath: "\"path\"to\"file\"",
+			LogRotateConfig: config.LogRotateConfig{
+				MaxFileSizeMB:   2,
+				BackupFileCount: 2,
+				Compress:        true,
+			},
+		},
+	}
+
+	actual := stringify(mountConfig)
+
+	expected := "{\"CreateEmptyFile\":false,\"Severity\":\"TRACE\",\"Format\":\"\",\"FilePath\":\"\\\"path\\\"to\\\"file\\\"\",\"LogRotateConfig\":{\"MaxFileSizeMB\":2,\"BackupFileCount\":2,\"Compress\":true}}"
+	AssertEq(strings.TrimSpace(expected), strings.TrimSpace(actual))
```

Comment:
Just curious, why do we need TrimSpace here?

Reply:
its not needed now actually, have removed it (added it earlier was testing random data without quotes)